pontoon/base/models/entity.py

```diff
@@ -889,8 +889,8 @@ def for_project_locale(
             # Search in source strings
             entity_filters = (
                 (
-                    Q(pk=None)  # Ensures that no source strings are returned
-                    if search_translations_only
+                    Q(pk__in=[])  # Ensures that no source strings are returned
```

Comment:
Why this change?

Reply:
I read over the [StackOverflow post](https://stackoverflow.com/questions/29900386/how-to-construct-django-q-object-matching-none) again, it seems that using `pk__in=[]` is a safer and potentially less costly alternative than `pk=None`

Reply:
Nice work with the research!